beacon-chain/core/helpers/BUILD.bazel

```diff
@@ -21,6 +21,7 @@ go_library(
     visibility = ["//visibility:public"],
     deps = [
         "//beacon-chain/cache:go_default_library",
+        "//beacon-chain/core/signing:go_default_library",
```

Comment:
These are the results of `bazel run //:gazelle -- fix`. We must not be running it up to date in `epbs` branch